src/crimson/os/seastore/seastore.cc

```diff
@@ -1489,16 +1489,26 @@ SeaStore::tm_ret SeaStore::_setattrs(
 {
   LOG_PREFIX(SeaStore::_setattrs);
   DEBUGT("onode={}", *ctx.transaction, *onode);
+
+  auto fut = tm_iertr::now();
```

Comment:
This commit seems to be pushed by mistake?

Reply:
No, it fixes a bug in seastore that made the last watcher unremovable.